packages/rest-specs/specs/resiliency/srv-driven/old.tsp

```diff
@@ -0,0 +1,119 @@
+import "@typespec/http";
```

Comment:
This folder too

Reply:
Removed the resiliency folder as mentioned above